behind-the-veil-siteroot/imports/ui/components/pages/artist/ArtistServiceArea.jsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * File Description: Service Area for artists within the settings page
+ * File version: 1.0
+ * Contributors: Hirun
+ */
+import React from "react";
+import WhiteBackground from "../../whiteBackground/WhiteBackground.jsx";
+import PageLayout from "../../../enums/PageLayout";
+import Tabs from "../../tabs/Tabs.jsx";
+import Button from "../../button/Button.jsx";
+import { ArrowLeftIcon, ArrowRightIcon } from '@heroicons/react/24/outline'
+
+/**
+ * This page allows the artist to enter to enter a location as well as provide a radius in which they can travel from that location
+ */
+
+export const ArtistServiceArea = () => {
+  const ServiceAreaPanel = (
+    <div className="grid grid-cols-12 gap-4">
```

Comment:
Looking at the design, you only need 3 columns instead of 12 columns. I see what you're trying to do, but doing it this way causes some issues (see picture). 
<img width="597" alt="Screenshot 2024-05-01 at 7 04 56 pm" src="https://github.com/Monash-FIT3170/Behind-The-Veil/assets/95511987/4827bd9c-1679-40cd-945d-988cceab8449">

If you want to continue using a grid, you can still define varying column widths in other ways, see: https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns

Here's a very rough example of something you can do (**this is just a demo, can use this as a start but i don't recommend just using it as is**). You can put this in your code and have a play, but basically, you can define custom styles by using square brackets in tailwind, if tailwind doesn't provide exactly what you're looking for.
"grid-cols-[5fr_1fr_1fr]" basically means, give me 3 columns, with a 5:1:1 ratio of width:
```    
<div className="grid grid-cols-[5fr_1fr_1fr] gap-4">
      <div>Location</div>
      <div>Radius (km)</div>
      <div></div>
      {/* row 1 */}
      <div>Melbourne CBD</div>
      <div>15</div>
      <div>TRASH ICON</div>
      {/* row 2 */}
      <div>Melbourne Airport</div>
      <div>3</div>
      <div>TRASH ICON</div>
      {/* inputs */}
      <input
        type="text"
        className="rounded outline outline-1 outline-light-grey p-2"
      />
      <input
        type="text"
        className="rounded outline outline-1 outline-light-grey p-2"
      />
      {/* use plus icon instead of + */}
      <Button>+ Add</Button>
</div>
```

which will give you something like this, but is more robust (esp when you resize the screen):
<img width="734" alt="Screenshot 2024-05-01 at 7 32 19 pm" src="https://github.com/Monash-FIT3170/Behind-The-Veil/assets/95511987/f8ec2beb-8a9e-401c-9bd3-a1ef649efc86">

Alternatively, you can use flex boxes to achieve this layout as well: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox